lib/getCosignMintCall.tsx

```diff
@@ -0,0 +1,25 @@
+import { Interface } from "ethers/lib/utils"
+import cosignAbi from "@/lib/abi/catalog-cosign.json"
+import { BigNumber } from "ethers"
+import { CATALOGCOSIGN_ADDRESS, CATALOG_ID, CATALOG_REFERRAL } from "./consts"
+import getCosignPrice from "./getCosignPrice"
+
+const getCosignMintCall = async (mintRecipient) => {
+  const tokenPrice = await getCosignPrice()
```

Comment:
we can `hard-code` cosign price
- same on `testnet` vs `mainnet` ✅ 
- same on `any` token ✅ 
- removes `async` requirement ⚡ 
```suggestion
  const tokenPrice = 1000000000000000n
```

Reply:
fixed ✅